relay-server/src/endpoints/nel.rs

```diff
@@ -45,18 +45,17 @@ async fn handle(
         return Ok(StatusCode::UNSUPPORTED_MEDIA_TYPE.into_response());
     }
 
-    let items: Value =
+    let items: Vec<Box<RawValue>> =
```

Comment:
Why does `RawValue` need to be boxed here? Could We do a `Vec<&RawValue>` instead?

Reply:
yeah. you're right, changed it